pages/common/httpx.md

```diff
@@ -0,0 +1,37 @@
+# httpx
+
+> A fast and multi-purpose HTTP toolkit written in Go that allows running multiple probes.
+> (Note that this is not the Python HTTPX which has the exact same command name).
+> More information: <https://github.com/projectdiscovery/httpx>.
+
+- Run a probe against a URL, host, IP Address or subnet (CIDR notation) showing probe status:
+
+`httpx -probe -u {{url|host|ipaddress|subnet_with_cidr}}`
+
+- Run a probe against multiple hosts showing [s]tatus [c]ode with input from `subfinder`:
+
+`subfinder -d {{example.com}} | httpx -sc`
+
+- Run a [r]ate [l]imited probe against a [l]ist of hosts from a file showing [t]echnology [d]etected and [r]esponse [t]ime:
+
+`httpx -rl {{150}} -l {{file_containing_newline_separate_hosts}} -td -rt`
+
+- Run a probe against a URL showing webpage title, CDN/WAF in use, and page content hash:
+
+`httpx -u {{url}} -title -cdn -hash {{sha256}}`
+
+- Run a probe against a list of hosts with custom defined [p]orts and timeout after certain seconds:
+
+`httpx -probe -u {{host1}},{{host2}},{{hostN}} -p http:{{80,8000-8080}},https:{{443,8443}} -timeout {{10}}`
+
+- Run a probe against a list of hosts [f]iltering out [c]odes of certain responses:
+
+`httpx -u {{host1}},{{host2}},{{hostN}} -fc {{400,401,404}}`
+
+- Run a probe against a list of hosts [m]atching [c]odes of certain responses:
+
+`httpx -u {{host1}},{{host2}},{{hostN}} -mc {{200,301,304}}`
+
+- Run a probe against a URL [s]aving [s]creenshots of certain paths, with [s]creenshot [t]imeouts (assets are saved in ./output):
```

Comment:
```suggestion
- Run a probe against a URL [s]aving [s]creenshots of certain paths, with [s]creenshot [t]imeouts (assets are saved in `./output`):
```
From the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases):
> Technical wording on description lines should use the `backtick` syntax.
> Use backticks on the following:
> 
> - Paths, e.g. `package.json`, `/etc/package.json`.
> - Extensions, e.g. `.dll`.
> - Commands, e.g. `ls`.
> - Standard streams: `stdout`, `stdin`, `stderr`. **Do not** use the full names (e.g. standard output).